src/tools/storage-key-recipe-resolver.ts

```diff
@@ -38,7 +38,7 @@ export class StorageKeyRecipeResolver {
    */
   async resolve(): Promise<Recipe[]> {
     const recipes = [];
-    for (const recipe of this.runtime.context.allRecipes) {
+    for (const recipe of this.runtime.context.recipes) {
```

Comment:
:+1: Good to know

Reply:
This will only generate plans for the explicitly provided manifest. I think that's what you want.

Reply:
probably irrelevant for now, but we have used the paradigm of "canonical" manifests that just import various manifests. do we explicitly don't want to support it here?

Reply:
In my estimation it would be clearer if we used the blaze / bazel deps + the same mechanisms as are used for code to drive what plans are compiled-in.

Reply:
We can talk more over chat maybe.

Reply:
no, that's fine, i just wanted to talk this through. agreed, deps are better for this. thanks.